shared/domains/chelonia/internals.js

```diff
@@ -292,14 +318,12 @@ export default (sbp('sbp/selectors/register', {
         console.warn(`[chelonia] publish attempt ${attempt} of ${maxAttempts} failed. Waiting ${randDelay} msec before resending ${entry.description()}`)
         attempt += 1
         await delay(randDelay) // wait randDelay ms before sending it again
-        // if this isn't OP_CONTRACT, recreate and resend message
-        if (!entry.isFirstMessage()) {
-          const rootState = sbp(this.config.stateSelector)
-          const newEntry = await recreateEvent(entry, rootState)
-          if (!newEntry) {
-            return
-          }
-          entry = newEntry
+
+        // TODO: The [pubsub] code seems to miss events that happened between
+        // a call to sync and the subscription time. This is a temporary measure
+        // to handle this until [pubsub] is updated.
+        if (entry.height() === lastAttemptedHeight) {
+          await sbp('okTurtles.eventQueue/queueEvent', entry.contractID(), ['chelonia/private/in/syncContract', entry.contractID()])
```

Comment:
So I agree that we should address this in this PR... one way to go about it is to make sure the pubsub is connected before `'chelonia/contract/sync'` does the sync.

We might also want to verify that we are in fact connected to the pubsub server before allowing messages to be sent.